doc/src/sgml/ref/pg_rewind.sgml

```diff
@@ -185,54 +255,86 @@ PostgreSQL documentation
  </refsect1>
 
  <refsect1>
+<!--
   <title>Environment</title>
+-->
+  <title>環境</title>
 
   <para>
-   When <option>--source-server</> option is used,
+<!--
+   When <option>#045;&#045;source-server</> option is used,
    <application>pg_rewind</application> also uses the environment variables
    supported by <application>libpq</> (see <xref linkend="libpq-envars">).
+-->
+<option>--source-server</>オプションを使用する場合、<application>pg_rewind</application>は
+<application>libpq</>で利用できる環境変数を使用します(<xref linkend="libpq-envars">を参照)。
   </para>
  </refsect1>
 
  <refsect1>
+<!--
   <title>Notes</title>
+-->
+  <title>注釈</title>
 
   <refsect2>
+<!--
    <title>How it works</title>
+-->
+   <title>どうやって動くのか</title>
 
    <para>
+<!--
     The basic idea is to copy everything from the new cluster to the old
     cluster, except for the blocks that we know to be the same.
+-->
+基本的なアイデアは、あらかじめ同じであることがわかっているブロックを除き、すべてを新しいクラスタから古いクラスタにコピーする、というものです。
    </para>
 
    <procedure>
     <step>
      <para>
+<!--
       Scan the WAL log of the old cluster, starting from the last checkpoint
       before the point where the new cluster's timeline history forked off
       from the old cluster. For each WAL record, make a note of the data
       blocks that were touched. This yields a list of all the data blocks
       that were changed in the old cluster, after the new cluster forked off.
+-->
+古いクラスタからタイムラインが新しいクラスタで分岐した時点より前の最後のチェックポイントから始めて、古いクラスタのWALログをスキャンしします。
+各々のWALレコードについて、変更されたデータブロックを記録します。
+これにより、新しいクラスタで分岐した以降に、古いクラスタで変更されたすべてのデータブロックのリストが作成できます。
      </para>
     </step>
     <step>
      <para>
+<!--
       Copy all those changed blocks from the new cluster to the old cluster.
+-->
+変更されたすべてのブロックを新しいクラスタから古いクラスタにコピーします。
      </para>
     </step>
     <step>
      <para>
+<!--
       Copy all other files such as <filename>clog</filename> and configuration files from the new cluster
       to the old cluster, everything except the relation files.
+-->
+<filename>clog</filename>や設定ファイルなど、リレーションファイル以外のすべてのファイルを新しいクラスタから古いクラスタにコピーします。
      </para>
     </step>
     <step>
      <para>
+<!--
       Apply the WAL from the new cluster, starting from the checkpoint
       created at failover. (Strictly speaking, <application>pg_rewind</>
       doesn't apply the WAL, it just creates a backup label file indicating
       that when <productname>PostgreSQL</> is started, it will start replay
       from that checkpoint and apply all the required WAL.)
+-->
+フェイルオーバの際に作られたチェックポイントから始めて、新しいクラスタのWALを適用します。(厳密に言うと、<application>pg_rewind</>はWALの適用はしません。
+<productname>PostgreSQL</>が起動した時を示すバックアップラベルファイルを作るだけです。
+それにより、チェックポイントから始めてすべての必要なWALが適用されます)
```

Comment:
ちょっと怪しげな原文で、rewriteを要求したいところですが、"indicating"の目的語はthatから文末まで全部です。
"that"がなければ「起動した時を示す」の意味になりますが、そうではないので、"when PostgreSQL is started"は"it will..."の方に掛かり、「…WALは適用せず、単にバックアップラベルファイルを作るだけです。それにより、PostgreSQLが起動するとき、そのチェックポイントから再生を初めて、すべての必要なWALを適用します」という感じの意味になります。